Document new OAuth scopes (#2014)
huggingface/hub-docs: docs/hub/spaces-oauth.md

- `email`: Get the user's email address.
- `read-billing`: Know whether the user has a payment method set up.
- `read-repos`: Get read access to the user's personal repos.

> - `contribute-repos`: Create repos, and have access to the repos created by this app. Cannot access other repos by default.

Author: ```suggestion
- `contribute-repos`: Create repos, and have access to the repos created by this app. Cannot access other repos (unless other permissions are granted).
```

> - `contribute-repos`: Create repos, and have access to the repos created by this app. Cannot access other repos (unless other permissions are granted).

Author: ```suggestion
- `contribute-repos`: Can create repositories and access those created by this app. Cannot access any other repositories unless additional permissions are granted.
```
